src/components/deploymentConfig/DeploymentTemplateView.tsx

```diff
@@ -416,10 +420,15 @@ const CompareWithDropdown = ({
             })
 
             if (!selectedOption) {
```

Comment:
In line 413 we are already using (!selectedOption) this check so can we merge these two conditions

Reply:
Can be merged but need to perform additional checks.

Reply:
Done... Re-wrote in simplified lines with comments.